stdlib/public/core/UInt128.swift

```diff
@@ -280,7 +280,9 @@ extension UInt128 {
   public func dividedReportingOverflow(
     by other: Self
   ) -> (partialValue: Self, overflow: Bool) {
-    _precondition(other != .zero, "Division by zero")
+    if _slowPath(other == .zero) {
+      return (self, true)
+    }
     // Unsigned divide never overflows.
```

Comment:
This comment seems ironic now, doesn't it?

Reply:
In a way, I suppose. I would say that the ironic part is the overflow return label, however, since it represents both proper overflow and another kind of failure (division by zero). Still, I can remove the comment if you find it unfitting. Generally, I don't mess with comments I did not write.

Reply:
It's fine to leave it, I think.